core/server/data/migrations/versions/3.22/02-settings-key-renames.js

```diff
@@ -0,0 +1,81 @@
+const logging = require('../../../../../shared/logging');
+
+const renameMapping = [{
+    from: 'default_locale',
+    to: 'lang'
+}, {
+    from: 'active_timezone',
+    to: 'timezone'
+}, {
+    from: 'ghost_head',
+    to: 'codeinjection_head'
+}, {
+    from: 'ghost_foot',
+    to: 'codeinjection_foot'
+}];
+
+module.exports = {
+    config: {
+        transaction: true
+    },
+
+    async up(options) {
+        await Promise.map(renameMapping, async (renameMap) => {
+            logging.info(`Renaming ${renameMap.from} to ${renameMap.to}`);
+
+            return await options
+                .transacting('settings')
+                .where('key', renameMap.from)
+                .update({
+                    key: renameMap.to
+                });
+        });
+
+        const brandResult = await options
+            .transacting('settings')
+            .where('key', 'brand')
+            .select('value');
+
+        const brand = JSON.parse(brandResult[0].value);
+
+        logging.info(`Updating brand.publicationColor in settings to accent_color with value ${brand.primaryColor}`);
```

Comment:
Minor logging issue but is `brand.publicationColor` meant to be `brand.primaryColor` here and in the logging line below?

Reply:
Good catch :fishing_pole_and_fish: will fix asap!